### ISSUE TITLE
enableBrCompress 4 inside httpd_config.conf

### ISSUE THREAD

**Issue by luboslives** [litespeedtech/openlitespeed#172]
The TLDR - this line causes v1.6.12 to crash for me. When I remove it, all is back to normal.

I recently updated from 1.5.11 to 1.6.12 using yum, and yum saves the new .conf files as conf.rpmnew to avoid overwriting – so I decided to take a look inside httpd_config to see what's different. I noticed this line, and I was actually hoping this indicated a dynamic Brotli compression level of 4 (which I had asked about in the official OLS forums a little while ago) so I gave it a shot, and upon graceful restart I could no longer access the admin panel. I removed the line and everything was fine.

For other reasons, I decided to do a fresh install of 1.6.12 and after it completed, I couldn't access the panel again at the default port 7080. My first instinct was to remove this enableBrCompress line from the freshly-installed config file, and voila, problem solved. The panel was loading again, after I accepted the new SSL cert.

So, what is this line for and why am I running into this problem when I see it's been in the core since v1.6.3?

**Comment by benyaminl** [comment]
> 
> 
> The TLDR - this line causes v1.6.12 to crash for me. When I remove it, all is back to normal.
> 
> I recently updated from 1.5.11 to 1.6.12 using yum, and yum saves the new .conf files as conf.rpmnew to avoid overwriting – so I decided to take a look inside httpd_config to see what's different. I noticed this line, and I was actually hoping this indicated a dynamic Brotli compression level of 4 (which I had asked about in the official OLS forums a little while ago) so I gave it a shot, and upon graceful restart I could no longer access the admin panel. I removed the line and everything was fine.
> 
> For other reasons, I decided to do a fresh install of 1.6.12 and after it completed, I couldn't access the panel again at the default port 7080. My first instinct was to remove this enableBrCompress line from the freshly-installed config file, and voila, problem solved. The panel was loading again, after I accepted the new SSL cert.
> 
> So, what is this line for and why am I running into this problem when I see it's been in the core since v1.6.3?

It's for Brotli Compression https://github.com/google/brotli from google that already widely used on web that can compress until 30% lower than gzip. Anyway if you wanted to use it with panel, I think you should use SSL so it will work, like HTTP2

(Hope I'm wrog).

**Comment by litespeedtech** [comment]
It should work properly, if still have problem please let us know.
